hdf5/examples/continuously_rw_1d.rs

```diff
@@ -0,0 +1,83 @@
+//! Create, and continuously write several chunks to a chunked 1D dataset
+//!
+//! Finally read it, get the chunking metadata, and use it to read slices aligned to the chunk size
+
+use hdf5::{File, Result};
+use hdf5_metno::{self as hdf5, Extent};
+use ndarray::Array1;
+
+// Shared between read_hdf5() and write_hdf5(), otherwise they only rely on file metadata
+const FILE_NAME: &str = "continuous_chunks.h5";
+const DATASET_NAME: &str = "count";
+
+const CHUNK_SIZE: usize = 5;
+const NUM_CHUNKS: usize = 3; // total chunks to write
+
+fn write_hdf5() -> Result<()> {
+    println!("Creating file '{FILE_NAME}' with 1D resizable dataset");
+    let file = File::create(FILE_NAME)?;
+    let shape = Extent::resizable(1); // 1D resizable
```

Comment:
```suggestion
    let shape = Extent::resizable(0); // 1D resizable
```
and drop line 22-25. Alternatives:
```rust
let shape = (0..,);
let shape = [0..];
let shape = 0..;
let shape = [Extent::resizable(0)];
```